NorthstarDLL/namedpipeclient.cpp

```diff
@@ -0,0 +1,140 @@
+#pragma once
+#include "pch.h"
+#include "namedpipeclient.h"
+#include "squirrel.h" // Squirrel
+
+// Named Pipe Stuff
+#include <windows.h>
+#include <conio.h>
+#include <tchar.h>
+#include <string>
+#include <atlstr.h>
+
+using namespace std;
```

Comment:
All symbols from namespace "std" are imported to the global scope, but only "string" and "exception" are used. Only declare used types, or access types from its namespace.